### DISCUSSION TITLE
Sample dataset

### DISCUSSION THREAD

**Discussion by Tima-Ze** [maclean-lab/RVAgene#7] [General] [answered]
Hi,
Thanks for developing such a useful method, I just guess that these two reference should be replaced with each other: "Bulk RNA seq Data used in the paper from : https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65525 (Klein et. al. 2015) scRNA seq Data used in the paper from : https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98622 (Liu. et. al. , 2017):
the first link includes single cell dataset and second one is a bulk RNAseq.

**Comment by timkartar** [comment, marked as answer]
You are right. Thanks for catching the typo. It's fixed now.
